content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/image.html

```diff
@@ -45,4 +45,8 @@
     <span class="cmp-image__title" itemprop="caption" data-sly-test="${!image.displayPopupTitle && image.title}">${image.title}</span>
     <meta itemprop="caption" content="${image.title}" data-sly-test="${image.displayPopupTitle && image.title}">
 </div>
-<sly data-sly-call="${templates.placeholder @ isEmpty = !image.src, classAppend = 'cmp-image cq-dd-image'}"></sly>
\ No newline at end of file
+<sly data-sly-call="${templates.placeholder @ isEmpty = !image.src, classAppend = 'cmp-image cq-dd-image'}"></sly>
+
+<script type="text/javascript">
```

Comment:
Would place the script before the placeholder call. Perhaps it could even be the last child of the component `div` ?